CMakeLists.txt

```diff
@@ -61,7 +61,7 @@ include_directories(${LUA_INCLUDE_DIR})
 find_library(LUA_LIBRARY lua)
 link_libraries(${LUA_LIBRARY})
 
-find_package(Doxygen)
+find_package(Doxygen QUIET)
```

Comment:
What's this do?

Reply:
It just prevents cmake from spewing a message about not being able to find Doxygen.

Doxygen is currently optional and we don't want it's non-existence failing CI.

Reply:
True. So what's the plan for Doxygen generation in our CI/CD process?

Reply:
Log an issue and figure out what's required to set it up on the CI machines. It will be easier once the initial support is merged as PR jobs can start running.

Reply:
Logged. Resolving and approving.